plugin.cfg

```diff
@@ -64,3 +64,4 @@ erratic:erratic
 whoami:whoami
 on:github.com/caddyserver/caddy/onevent
 sign:sign
+fanout:fanout
```

Comment:
IMO, this should be moved to immediately after forward.

Reply:
thanks, fixed